### DISCUSSION TITLE
range

### DISCUSSION THREAD

**Discussion by gkim132** [jspsych/jsPsych#1209] [Q&A] [answered]
Hi,

I'm wondering if there is a jspsych plugin that can save the outcome of <input type="range"> which is an HTML element [.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) 

I appreciate any other suggestions as well.

Any help would be appreciated!

Thank you!

**Comment by becky-gilbert** [comment, marked as answer]
Hi @gkim132, jsPsych has a few different slider (input type="range) response plugins: [audio-slider-response](https://www.jspsych.org/plugins/jspsych-audio-slider-response/), [html-slider-response](https://www.jspsych.org/plugins/jspsych-html-slider-response/), [image-slider-response](https://www.jspsych.org/plugins/jspsych-image-slider-response/), and [video-slider-response](https://www.jspsych.org/plugins/jspsych-video-slider-response/). There are example HTML files for each in the jsPsych /examples folder. Do these do what you want?

Also, if you want to display the current value while the participant moves the slider, you might want to check out @gorkang's [modified slider plugin](https://github.com/jspsych/jsPsych/issues/1192) that contains a `slider_number` true/false parameter.
